src/header/navigation.rs

```diff
@@ -23,17 +23,9 @@ pub fn Header() -> impl IntoView {
                     ..Default::default()
                 },
                 NavEntry {
-                    href: "/transactions/token".to_string(),
-                    text: "Token Transactions".to_string(),
-                    icon: NavIcon::Tokens,
-                    disabled: true,
-                    ..Default::default()
-                },
-                NavEntry {
-                    href: "/transactions/zkapp".to_string(),
+                    href: "/transactions/zk-trx".to_string(),
```

Comment:
Industry standard is to abbreviate "transaction" as "txn", FYI.

Reply:
Ok, I will refactor in another issue, as the change is much broader than this file.